nupic/__init__.py

```diff
@@ -19,4 +19,5 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
+__version__ = "0.2.7.dev0"
```

Comment:
You can't have anything other than the namespace declaration in this file since this is a namespace package. [See here](https://pythonhosted.org/setuptools/setuptools.html#namespace-packages).